src/applications/static-pages/download-1095b/tests/01-1095b-authed.cypress.spec.js

```diff
@@ -25,8 +25,8 @@ describe('Authed 1095-B Form Download PDF', () => {
 
     cy.axeCheck();
 
-    cy.get('#1095-download-options-0').should('be.checked');
-    cy.get('#1095-download-options-1').should('not.be.checked');
+    cy.get('#pdf').should('be.visible');
+    cy.get('#txt').should('be.visible');
 
```

Comment:
@ryguyk / @allisonlu / @jtmst , this just updates the spec file to what you already have in `main` now to get CI to pass without me having to do a full merge and rebase.

Reply:
Cool. I'm approving accordingly. I haven't looked over the rest of the PR.